components/vc/front/SConscript

```diff
@@ -158,6 +160,7 @@ project_source_files = [
     SRC_DIR.File("brakePressure.c"),
     SRC_DIR.File("steeringAngle.c"),
     SRC_DIR.File("shockpot.c"),
+    SHARED_VC.File("wheelSpeed.c"),
```

Comment:
keeping these alphabetized will help if we switch to buck

Reply:
actually I take it back. alphabetization doesn't matter, we just need to preserve the order when we re-implement this in buck. We will for sure have to re-order some stuff in scons in order to make it hex-neutral vs. buck though, but we can do that later